docs/tasks/access-kubernetes-api/setup-api-aggregator.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Setup the API Aggregator
+assignees:
+- chenopis
+---
+
+{% capture overview %}
+
+Setting up the API Aggregator will allow the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* You need to have a Kubernetes cluster running.
+* Make sure there are Aggregator certificates (both CA and proxy) setup on the Kubernetes apiserver.
+* The proxy cert must have been signed by the relevant CA. 
```

Comment:
Kubernetes (and the kube-apiserver) have multiple CAs. I want to make sure the proxy is signed by the aggregator CA and not say the master CA.